.github/workflows/cdelivery-s3-apps.yml

```diff
@@ -81,9 +83,9 @@ jobs:
       # Set environment variables required to perform the build. These are only available to this step
       env:
         REACT_APP_GRAASP_DOMAIN: ${{ secrets.graasp-domain }}
-        REACT_APP_GRAASP_APP_ID: ${{ secrets.app-id }}
+        REACT_APP_GRAASP_APP_KEY: ${{ secrets.app-key }}
         REACT_APP_SENTRY_DSN: ${{ secrets.sentry-dsn }}
-        REACT_APP_MOCK_API: false
+        REACT_APP_ENABLE_MOCK_API: false
```

Comment:
Nice, did we update the docs of the apps-query-client and/or of the apps to reflect this change ?